cirq-google/cirq_google/api/v2/sweeps_test.py

```diff
@@ -98,6 +101,20 @@ def test_sweep_to_proto_linspace():
     )
 
 
+@pytest.mark.parametrize("val", [None, 1, 1.5, 's'])
+def test_build_recover_const(val):
+    val2 = v2.sweeps._recover_sweep_const(v2.sweeps._build_sweep_const(val))
+    if isinstance(val, float):
+        assert math.isclose(val, val2)
```

Comment:
nit: can you add a comment explaining why using `isclose` is necessary ?

Reply:
Done